File: src/android/PhotoLibraryGetLibraryOptions.java
package com.terikon.cordova.photolibrary;

public class PhotoLibraryGetLibraryOptions {

  public final int itemsInChunk;
  public final int maxItems;
  public final double chunkTimeSec;
  public final boolean includeAlbumData;

  public PhotoLibraryGetLibraryOptions(int itemsInChunk, int maxItems, double chunkTimeSec, boolean includeAlbumData) {
    this.itemsInChunk = itemsInChunk;
    this.maxItems = maxItems;
    this.chunkTimeSec = chunkTimeSec;
    this.includeAlbumData = includeAlbumData;
  }

}

File: src/ios/PhotoLibraryGetLibraryOptions.swift
import Foundation

struct PhotoLibraryGetLibraryOptions {
    let thumbnailWidth: Int
    let thumbnailHeight: Int
    let itemsInChunk: Int
    let chunkTimeSec: Double
    let useOriginalFileNames: Bool
    let includeImages: Bool
    let includeAlbumData: Bool
    let includeCloudData: Bool
    let includeVideos: Bool
    let maxItems: Int
}
